commands/kubernetes.go

```diff
@@ -1721,6 +1721,9 @@ func parseNodePoolString(nodePool, defaultName, defaultSize string, defaultCount
 		Taints: []godo.Taint{},
 	}
 	for _, arg := range strings.Split(nodePool, argSeparator) {
```

Comment:
How about trimming any trailing `;`?

```go
	trimmedPool := strings.TrimSuffix(nodePool, argSeparator)
	for _, arg := range strings.Split(trimmedPool, argSeparator) {
```

It's a bit of a corner case, but it provides a proper error if someone were to run:

    doctl kubernetes cluster create test --node-pool ";"

Reply:
That seems reasonable. Thanks for the review, I have now modified the code.